lockbox-ios/Common/Extensions/UIImage+.swift

```diff
@@ -0,0 +1,37 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import UIKit
+
+extension UIImage {
+    static func createGradientImage(frame:CGRect, colors: [UIColor]) -> UIImage? {
+        let gradientLayer = CAGradientLayer()
+
+        gradientLayer.colors = colors.map({ (color) -> CGColor in
+            return color.cgColor
+        })
+        gradientLayer.frame = frame
+        gradientLayer.locations = [0.0, 0.85]
```

Comment:
Yeah, it could be parameterized... it's pretty specific to the design for the gradient that I was given so maybe I could parameterize & address with the other constant aggregation

Reply:
O K A Y :)